openfl-tutorials/interactive_api/PyTorch_BrainMRI_UNet/workspace/BrainMRI2D_Segmentation_OpenFL_Tutorial.ipynb

```diff
@@ -0,0 +1,476 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "893aa064",
+   "metadata": {},
+   "source": [
+    "## Medical Decathlon BrainMRI Segmentation OpenFL tutorial"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "120c95ec",
+   "metadata": {},
+   "source": [
+    "### Download the dataset : Medicat Decathlon BrainMRI Segmentation Dataset\n",
+    "\n",
+    "Download the data from : https://goo.gl/QzVZcm download the file Task01_BrainTumour.tar (into the base repository) <p>\n",
+    "Then run the prepare_dataset.py file to process and create the dataset so that it is ready to use. <p>\n",
```

Comment:
The instructions to download the dataset/setup etc should be a part of README and removed from the notebook.